cmake/externals.cmake

```diff
@@ -168,15 +168,16 @@ if (INCLUDE_DOCKER_PLUGINS)
 endif()
 
 if (INCLUDE_MOZSVC)
-    add_external_plugin(git https://github.com/mozilla-services/heka-mozsvc-plugins 848fc1f3aa858472150c7af5463661393d3c4f3b)
+    #add_external_plugin(git https://github.com/mozilla-services/heka-mozsvc-plugins 848fc1f3aa858472150c7af5463661393d3c4f3b)
+    add_external_plugin(git https://github.com/mozilla-services/heka-mozsvc-plugins :local)
```

Comment:
yup, good catch. i'll push my commit to heka-mozsvc-plugins repo and will get this fixed.